stage0/run.go

```diff
@@ -60,19 +61,19 @@ type PrepareConfig struct {
 // configuration parameters needed by Run
 type RunConfig struct {
 	CommonConfig
-	PrivateNet           bool // pod should have its own network stack
-	SpawnMetadataService bool // launch metadata service
-	LockFd               int  // lock file descriptor
-	Interactive          bool // whether the pod is interactive or not
+	PrivateNet           bool         // pod should have its own network stack
+	SpawnMetadataService bool         // launch metadata service
+	LockFd               int          // lock file descriptor
+	Interactive          bool         // whether the pod is interactive or not
+	Images               []types.Hash // application images (prepare gets them via Apps)
```

Comment:
It can, but Apps implies so much more than just image ids is being expressed.  Run just needs the image ids because of overlayfs, it doesn't apply anything attached to Apps like prepare does.  I didn't want to create a false expectation in some future reader of this code that manipulating crap in Apps attached to the influences Run, because it doesn't, not right now anyways.